examples/ansible/setup_grafana.yml

```diff
@@ -0,0 +1,49 @@
+---
+# Ansible Playbook: Setup Grafana on SUSE ALP Dolomite
+# Description: This Ansible playbook automates the deployment of Grafana on a SUSE ALP Dolomite host.
+# The steps include fetching the Grafana image, setting up the Grafana container, and providing access information.
+# The steps are based on https://documentation.suse.com/alp/dolomite/html/alp-dolomite/available-alp-workloads.html#task-run-grafana-with-podman
+
+- name: Setup ALP system for Grafana
+  hosts: alphost
+  become: true
```

Comment:
Why would you need to be root for that?

Reply:
You do not need to be root to create variable in Ansible.
Variables are created locally, the `ansible.builtin.vars` plugin does not even connect to the host.

Reply:
Have mentioned 'become: true', at the playbook level for it to apply to all tasks.

Reply:
No, just keep it there.